typegraph/core/src/typegraph.rs

```diff
@@ -245,6 +235,15 @@ pub fn finalize(
         Err(e) => return Err(e),
     };
 
+    // TODO: maybe remove this extra return value? tg.meta.ref_artifacts can be accessed on the sdk frontend
```

Comment:
you mean by accessing it from the TgContext or from the serialized typegraph itself?

Reply:
yes, accessing it directly from the the serialized typegraph unless there are use cases that I am missing.

Reply:
I talked about this with @Natoandro and we thought rather than accessing it from the serialized typegraph, returning selectively the things you need as finalization result. We dropped the former approach not to introduce functionalities to parse the tgJson to typegraph object to the SDK.